guides/doc-Deploying_Project_on_AWS/topics/AWS-registering-hosts.adoc

```diff
@@ -0,0 +1,10 @@
+[[aws-registering-hosts]]
+= Registering Hosts to {Project}
+
+ifdef::foreman-el,katello[]
+This procedure applies only to users of the Katello plug-in.
+endif::[]
+
+When you install {ProjectServer} and {SmartProxyServer}, you must then register the content hosts on EC2 instances to {Project}.
```

Comment:
@ekohl Should we consider the navigation path *Hosts* > *Content Hosts* discouraged at this point and always navigate through *Hosts* > *All Hosts*?

Reply:
There are a couple reasons to still use the content hosts pages. Firstly, our "bulk manage packages/errata" workflows aren't moved over to the main Hosts page yet. Also, if you aren't using Simple Content Access, subscription management still needs to happen on a content host page.

So, overall, we are hoping that most users will not need to look at the old (legacy) content host pages. I suppose we could go as far to say that Hosts > Content Hosts is discouraged.

@jeremylenz do you agree?

Reply:
Yes, Hosts > Content Hosts at this point is only used for bulk actions.

Even for subscription actions, if you try to use Hosts > Content Hosts and click on a host, it will take you to the new host details page for that host (unless you've got that setting turned off), and then you'll have to use the "Legacy Content host UI" link to get to the old page that has the Subscriptions tab. So, may as well just use Hosts > All Hosts for everything at this point, except content host bulk actions.